admin/index.html

```diff
@@ -48,7 +48,7 @@ <h2>
                     <option value="2">Admin</option>
                 </select>
             </div>
-            <button id="login">
+            <button id="add-user-button">
```

Comment:
I tried not to make any HTML/CSS changes this time but this is a small exception